audit/projects/k8s-staging-cpa/services/enabled.txt

```diff
@@ -1,7 +1,6 @@
 NAME                              TITLE
 cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
-containeranalysis.googleapis.com  Container Analysis API
```

Comment:
Expected, this was mid-way through its scan of projects when I ran https://github.com/kubernetes/k8s.io/issues/1963#issuecomment-916309207